go.sum

```diff
@@ -1,3 +1,5 @@
+github.com/DataDog/zstd v1.5.2 h1:vUG4lAyuPCXO0TLbXvPv7EB7cNK1QV/luu55UHLrrn8=
+github.com/DataDog/zstd v1.5.2/go.mod h1:g4AWEaM3yOg3HYfnJ3YIawPnVdXJh9QME85blwSAmyw=
```

Comment:
This seems right to me, but checking to make sure we decided this on purpose: it seems like there is a pure Go zstd, while this is a wrapper around the C code.  That's ok with me, just wondering if we compared?